src/main/java/seedu/address/ui/StatusBarFooter.java

```diff
@@ -17,13 +22,62 @@
     @FXML
     private Label saveLocationStatus;
```

Comment:
Should change the name of this variable to better reflect its purpose

Reply:
Thanks for pointing out, I've changed to saveDateTime

Reply:
actually it just occured to me after merging that this name is also not appropriate LOL. It should be more like dateTimeFooter or something, because you aren't saving anything